src/components/RichTextEditor/Plugins/MentionPlugin/helpers/filterItems.ts

```diff
@@ -0,0 +1,6 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { TComboboxItemBase } from '@udecode/plate';
+
+export const filterItems = (value: string) => (item: TComboboxItemBase) =>
+    item.text.toLowerCase().search(new RegExp(`${value}`)) > -1;
```

Comment:
Isn't it the same as
```
export const filterItems = (value: string) => (item: TComboboxItemBase) =>
    item.text.toLowerCase().includes(value);
```
?

Reply:
I dont need the toLowerCase() and I can add flag to Regex, would guess this faster then transform and compare. What do you think?

Reply:
Then also value needs to be lowercase to work